src/styles/annotator/components/Buckets.scss

```diff
@@ -1,48 +1,107 @@
 
-.Buckets__bucket {
-  position: absolute;
-  right: 0;
-}
+    // Establish :before and :after content for later manipulation into
+    // different pointer shapes and directions
+    &::before,
+    &::after {
+      @apply content-[""] absolute border-transparent;
+      // NB: use of 'inset' here fixes jagged diagonals in FF
+      // https://github.com/zurb/foundation/issues/2230
+      // Note: there is no tailwind utility class for inset borders
+      border-style: inset;
+    }
+  }
 
-.Buckets__button {
-  // Need pointer events again. Necessary because of `pointer-events` rule
-  // in `.Buckets__list`
-  pointer-events: all;
-}
+  // Style a `BucketButton` to point left
+  .LeftBucketButton {
+    @apply rounded-r rounded-l-sm;
 
-.Buckets__button--left {
-  // Center the indicator vertically (the element is 16px tall)
-  margin-top: -8px;
-  @include buttons.indicator--left;
-}
+    // Position to the left of the button and centered vertically
+    &::before,
+    &::after {
+      @apply right-full top-1/2;
+    }
 
-.Buckets__button--up {
-  @include buttons.indicator--up;
-  // Vertically center the element (which is 22px high)
-  margin-top: -11px;
-}
+    // Create a grey wedge to the left of the button
+    // This will appear as the border around the pointy side
+    &::before {
+      @apply mt-[-8px] border-8;
+      @apply border-r-[5px] border-r-grey-3;
+    }
+
+    // Create a white wedge to the left of the button, 1px smaller than
+    // grey wedge. This will composite on top of the grey wedge and serve as
+    // the white fill of the pointy side
+    &::after {
+      @apply mt-[-7px] border-[7px];
+      @apply border-r-[4px] border-r-white;
+    }
+  }
+
+  // Style a `BucketButton` to point up
+  .UpBucketButton {
+    // Z-index assures that left-pointing buttons will scroll behind this
+    @apply z-1 rounded-t-sm rounded-b;
+
+    // Position above the button and horizontally centered
+    &::before,
+    &::after {
+      @apply top-auto left-1/2 bottom-full;
+    }
+
+    // Create a grey wedge at the top of the button
+    // This will appear as the border around the pointy side
+    &::before {
+      @apply ml-[-13px] border-solid border-[13px];
+      @apply border-b-[6px] border-b-grey-3;
+    }
+
+    // Create a white wedge 1px smaller than the grey wedge, as "fill"
+    &::after {
+      @apply ml-[-12px] border-solid border-[12px];
+      @apply border-b-[5px] border-b-white;
+    }
+  }
+
+  // Style a `BucketButton` to point down
+  .DownBucketButton {
+    // Z-index assures that left-pointing buttons will scroll behind this
+    @apply z-1 rounded-t rounded-b-sm;
+
+    // Position below the button and horizontally centered
+    &::before,
+    &::after {
+      @apply top-full left-1/2;
+    }
+
+    // Create a grey wedge at the bottom of the button
+    // This will appear as the border around the pointy side
+    &::before {
+      @apply ml-[-13px] border-[13px] border-solid;
```

Comment:
There are a number of rules shared by both the up- and down- button styling. I experimented with abstracting this into an abstract CSS class, `.VerticalBucketButton` or similar, but I felt that the inheritance and abstraction complexity outweighed the desire to have everything perfectly DRY. It's a balancing act. Having it unrolled allows one to be able to see it in one place.